src/plot/series/custom-svg-series.js

```diff
@@ -99,7 +99,7 @@ class CustomSVGSeries extends AbstractSeries {
     if (animation) {
       return (
         <Animation {...this.props} animatedProps={ANIMATED_SERIES_PROPS}>
-          <CustomSVGSeries {...this.props} animation={null}/>
+          <CustomSVGSeries {...this.props} animation={false}/>
```

Comment:
not providing the prop will do as if it was `undefined` and the condition would not be met anyway right?